### DISCUSSION TITLE
Academic domain verification

### DISCUSSION THREAD

**Discussion by doralhy-proteindesign** [RosettaCommons/rosetta#105] [ROSIE] [answered]
Hi ROSIE team,
Could you help to verify my email address ending with @lji.org, La Jolla Institute for Immunology (www.lji.org)?
We are a non-profit institution. 
Thank you!

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know, - this should be fixed now. Please feel free to re-open if you still having issues.
